newsfragments/3563.minor

```diff
@@ -0,0 +1 @@
+detect all port 0 configs
```

Comment:
My understanding of the convention is that if it's minor enough to merit a `.minor`, you can just leave the file empty. And it's meaningful enough to have a description, should maybe be `.bugfix` and be a more fleshed out English sentence that's meaningful to users. This seems pretty minor, so I'd do the former.

Reply:
Thanks noted. Do I have to undo this in this current PR?

Reply:
Probably fine?

Reply:
Great removing that